website/docs/blog.mdx

```diff
@@ -220,7 +220,6 @@ This option works best to get started, or for casual, irregular authors.
 
 Prefer usage of the `authors` FrontMatter, but the legacy `author_*` FrontMatter remains supported:
 
-<!-- prettier-ignore-start -->
```

Comment:
this is not needed anymore?

Reply:
No, the new prettier version fixed the extra empty line issue, part of the reason why I find the bump good.